cmd/joy/release.go

```diff
@@ -914,3 +919,55 @@ func NewReleaseLinksCmd() *cobra.Command {
 
 	return cmd
 }
+
+func NewReleaseSchemaCmd() *cobra.Command {
+	var env string
+	cmd := &cobra.Command{
+		Use:  "schema",
+		Args: cobra.RangeArgs(0, 1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				_, err := fmt.Fprintln(cmd.OutOrStdout(), v1alpha1.ReleaseSpecification())
+				return err
+			}
+
+			cat := catalog.FromContext(cmd.Context())
+
+			release, err := cat.LookupRelease(env, args[0])
+			if err != nil {
+				return fmt.Errorf("looking up release: %w", err)
+			}
+
+			cfg := config.FromContext(cmd.Context())
+
+			charts := helm.ChartCache{
+				Refs:            cfg.Charts,
+				DefaultChartRef: cfg.DefaultChartRef,
+				Root:            cfg.JoyCache,
+				Puller:          helm.CLI{IO: internal.IoFromCommand(cmd)},
+			}
+
+			chart, err := charts.GetReleaseChartFS(cmd.Context(), release)
+			if err != nil {
+				return fmt.Errorf("getting release chart: %w", err)
+			}
+
+			schemaBytes, err := chart.ReadFile("values.cue")
+			if err != nil {
+				if os.IsNotExist(err) {
+					_, err := io.WriteString(cmd.OutOrStdout(), v1alpha1.ReleaseSpecification())
+					return err
+				}
+				return fmt.Errorf("reading schema: %w", err)
+			}
+
+			schema := cuecontext.New().CompileBytes(schemaBytes).LookupPath(cue.MakePath(cue.Def("values")))
+
+			_, err = fmt.Fprintln(cmd.OutOrStdout(), internal.StringifySchema(schema))
+			return err
+		},
+	}
+
+	cmd.Flags().StringVar(&env, "env", "qa", "environment to find release from")
```

Comment:
"qa" seems arbitrary (and nesto-specific). Could either make it a required flag or default to "" and prompt user?

Reply:
Good catch. I forgot about the nesto-specificness of this for a hot minute. Yes, i will make it required if a release is present